### PR TITLE
Onboarding cleanup + misc changes

### PR DESCRIPTION
**Pull request by morebrownies**
**Changes**
- Fixed checkboxes in onboarding
- Fixed search icons in inputs in onboarding
- Made scrollbars a bit wider
- Bumped up the text size in the Discover toggle
- Typography fixes on the become a store modal

### CODE REVIEW

File: js/templates/home.html [hunk 1]
```diff
@@ -49,9 +49,9 @@ <h1 class="fancy-heading fontSize20 vCentered js-discoverHeading"><%= polyglot.t
                 <span class="ion-person fontSize11 marginRight2 toggle-txt"></span>
                 <span class="toggle-txt"><%= polyglot.t('ShowOnlyFollowerListings') %></span>
               </a>
-              <% if(!localStorage.getItem('safeListingsWarningDissmissed')){ %>
```
Comment (jjeffryes) [inline js/templates/home.html:52]: Why did you remove this? If you do that, the discover helper will appear when it shouldn't.

Comment (morebrownies) [inline js/templates/home.html:52]: ah, was testing without it and forgot to re-enable it.
